src/services/peopleSearch.ts

```diff
@@ -75,16 +76,27 @@ export type PeopleSearchQuery = {
   involvement: string;
 };
 
+const getRenamedDepartments = async () => {
+  const [dep] = await Promise.all([getDepartments()]);
+  dep.map((d, i) => {
+    if (/^Office of /.test(d)) {
+      dep[i] = dep[i].replace(/^Office of /, '') + ' (Office of)';
+    } else if (/^Center for /.test(d)) {
+      dep[i] = dep[i].replace(/^Center for /, '') + ' (Center for)';
+    } else if (/^Department of /.test(d)) {
+      dep[i] = dep[i].replace(/^Department of /, '') + ' (Department of)';
+    }
+  });
+  return dep;
+};
```

Comment:
We can simplify this function a little bit more:
1. `Promise.all()` is only needed when you want to await a list of promises in parallel. To await one Promise, you do not need it:
```suggestion
const getRenamedDepartments = async () => {
  const dep = await getDepartments();
  dep.map((d, i) => {
    if (/^Office of /.test(d)) {
      dep[i] = dep[i].replace(/^Office of /, '') + ' (Office of)';
    } else if (/^Center for /.test(d)) {
      dep[i] = dep[i].replace(/^Center for /, '') + ' (Center for)';
    } else if (/^Department of /.test(d)) {
      dep[i] = dep[i].replace(/^Department of /, '') + ' (Department of)';
    }
  });
  return dep;
};
```
2. `Array.map()` constructs a new array with the same length as the original, where each element is the result of the callback. For example, `const squares = [1, 2, 3].map(x => { return x * x; })` sets `squares` to `[2, 4, 6]`. So the way you're currently using it is wasteful because you're mutating the original array `dep` while also creating a new array of the same length that is filled with `undefined` elements (since a function/callback that doesn't return anything implicitly returns `undefined` in Javascript). The two ways to do this are to either:
   1. Use `Array.forEach` instead, which will loop through the elements with constructing a new array, so you can mutate the original array with no waste.
   2. Use `map` as intended. This way has some upsides because you don't have to worry about indices and mutation, you just declare the new value from the old value, which makes it much less error prone. It is technically less performant, but the difference is irrelevant unless you're mapping millions of elements constantly.
```suggestion
const getRenamedDepartments = async () => {
  const dep = await getDepartments();
  const updatedDeps = dep.map((d) => {
    if (/^Office of /.test(d)) {
      return d.replace(/^Office of /, '') + ' (Office of)';
    } else if (/^Center for /.test(d)) {
      return d.replace(/^Center for /, '') + ' (Center for)';
    } else if (/^Department of /.test(d)) {
      return d.replace(/^Department of /, '') + ' (Department of)';
    }
  });
  return updatedDeps;
};
```
or more succinctly
```suggestion
const getRenamedDepartments = async () => {
  const dep = await getDepartments();
  return dep.map((d) => {
    if (/^Office of /.test(d)) {
      return d.replace(/^Office of /, '') + ' (Office of)';
    } else if (/^Center for /.test(d)) {
      return d.replace(/^Center for /, '') + ' (Center for)';
    } else if (/^Department of /.test(d)) {
      return d.replace(/^Department of /, '') + ' (Department of)';
    }
  });
};
```
3. You could, if you want, use `.then` instead of `await`:
```suggestion
const getRenamedDepartments = () =>
  getDepartments().then(dep => dep.map((d) => {
    if (/^Office of /.test(d)) {
      return d.replace(/^Office of /, '') + ' (Office of)';
    } else if (/^Center for /.test(d)) {
      return d.replace(/^Center for /, '') + ' (Center for)';
    } else if (/^Department of /.test(d)) {
      return d.replace(/^Department of /, '') + ' (Department of)';
    }
  }));
```
And you can use the `map` utility function in `src/services/utils.ts` to make this slightly less verbose:
```suggestion
const getRenamedDepartments = () =>
  getDepartments().then(map((d) => {
    if (/^Office of /.test(d)) {
      return d.replace(/^Office of /, '') + ' (Office of)';
    } else if (/^Center for /.test(d)) {
      return d.replace(/^Center for /, '') + ' (Center for)';
    } else if (/^Department of /.test(d)) {
      return d.replace(/^Department of /, '') + ' (Department of)';
    }
  }));
```
4. You could actually collapse all three cases into one only slightly more complicated regular expression:
```suggestion
// Match any department whose name starts with one of the prefixes "Office of ", "Department of " or "Center for "
const departmentPrefixRegex = /^(Office of|Department of|Center for) (.*)$/;
const getRenamedDepartments = () =>
  // Match known prefixes and, if matched, move the prefix to the end of the department name in parentheses
  getDepartments().then(map((department) => department.replace(departmentPrefixRegex, "$2 ($1)"));
```

Reply:
For some reason the .map without dep[i] = dep[i] is not working, so I changed for the forEach. 

Also when using the big regular exp /^(Office of|Department of|Center for) (.*)$/ and then trying to replace(departmentPrefixRegex, "$2 ($1)")); is also not working, even making more changes to the syntax. Is it ok to not be that concise?